faqs.rst

```diff
@@ -64,26 +64,27 @@ Memory exhaustion on a node when running large ensembles with Flux
 
 Flux's in-memory KVS, or more properly, its content-addressable storage
 subsystem, is backed by an `SQLite <https://www.sqlite.org>`_ database file,
-located by default in ``/tmp`` (or ``$TMPDIR``).  On some systems, ``/tmp``
-is a RAM-backed file system.  To minimize Flux's memory footprint on such
-systems, Flux may be launched with the database file redirected to a more
-appropriate location.  For example:
+located by default in *rundir*, which is usually in ``/tmp``.  On some systems,
+``/tmp`` is a RAM-backed file system.  To minimize Flux's memory footprint
+on such systems, Flux may be launched with the database file redirected to
+a more appropriate location by setting the *statedir* broker attribute.  For
+example:
```

Comment:
Though the current wording makes sense, I keep worrying that the implicit statement that `statedir` overrides `rundir` for the database file should be made explicit. However, I'm having a hard time crafting alternative wording that isn't more clumsy than what you have here. This is the best I could come up with, take or leave it, it probably isn't a major concern.

```suggestion
located in *rundir* by default unless the *statedir* broker attribute is set.
Because *rundir*, which is usually in ``/tmp``, may point to a RAM-backed
filesystem, it may be advisable in some cases to launch Flux with the database
in a more appropriate location by setting *statedir*.
example:
```

Reply:
Hmm, I'm not having any luck coming up with an alternative wording either.  There is a link to the longer description in flux-broker-attributes(7) so I suppose people can easily dig a little deeper.  I'd say let's merge this and we can always submit another PR if better wording comes to mind.